Code/Framework/AzToolsFramework/AzToolsFramework/PaintBrushSettings/PaintBrushSettingsWindow_Internals.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <QListView>
+#include <AzToolsFramework/Entity/EditorEntityContextBus.h>
+#include <AzToolsFramework/API/ToolsApplicationAPI.h>
+#include <AzCore/Component/ComponentApplicationBus.h>
+#include <AzToolsFramework/Manipulators/PaintBrushManipulator.h>
+#include <AzToolsFramework/UI/PropertyEditor/PropertyEditorAPI_Internals.h>
+
+namespace AZ
+{
+    class SerializeContext;
+} // namespace AZ
+
+namespace AzToolsFramework
+{
+    class ReflectedPropertyEditor;
+} // namespace AzToolsFramework
+
+namespace PaintBrush
+{
+    namespace Internal
+    {
+        //! PaintBrushSettingsWindow is a simple view pane that lets us view and edit the global paint brush settings.
+        //! This pane is only visible while in a painting component mode.
+        //! Unlike other component modes, this is built as a separate pane because the controls might get fairly complex over time
+        //! and will likely get used frequently while painting, so the user should have the ability to move and dock these settings
+        //! to wherever is best for their paint session.
+        class PaintBrushSettingsWindow
+            : public QListView
+            , private AzToolsFramework::IPropertyEditorNotify
+        {
+        public:
+            PaintBrushSettingsWindow(QWidget* parent = nullptr);
+            ~PaintBrushSettingsWindow();
```

Comment:
Nit: The definition was empty, so could we just do this instead?
```suggestion
            ~PaintBrushSettingsWindow() = default;
```

Reply:
Whoops, good catch! It used to have code in it in a previous version, and I forgot to switch it when I removed the code. If there are any other changes requested from other reviewers, I'll switch this, otherwise I'll leave it for a subsequent PR.

Reply:
Actually, to fix one of the other pieces of feedback below, I need to put code back into the destructor, so I won't make this change. :)